.gitignore

```diff
@@ -14,3 +14,4 @@ db.sqlite3
 */setup.cfg
 
 .vscode
+/backend/media/
```

Comment:
Probably you need to change to `backend/media` as the slash `/` at the beginning will mean that it's from root.